### PR TITLE
Add additional comments for better description of battery property

### PR DESCRIPTION
**Pull request by sgchoi5**
Added some description using remarks TAG.

### CODE REVIEW

**Review by sgchoi5 — COMMENTED** [review]
LGTM

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@sgchoi5 I have reviewed the PR and added my feedback comments. Kindly check and update.

File: docs/application/native/api/wearable/6.0/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html [hunk 1]
```diff
@@ -251,6 +251,7 @@ <h2><a class="anchor" id="CAPI_SYSTEM_DEVICE_BATTERY_MODULE_OVERVIEW"></a>
 
 <p>Enumeration for battery property information. </p>
 <dl class="user"><dt><b>Since :</b></dt><dd>3.0 </dd></dl>
+<dl class="remark"><dt><b>Remarks:</b></dt><dd>If fuel gauge hardware does not provide average current or average voltage, CURRENT_AVERAGE/VOLTAGE_AVERAGE can be substituted by CURRENT_NOW/VOLTAGE_NOW, respectively. </dd></dl>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/api/wearable/6.0/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html]: Thanks!, fixed it

File: docs/application/native/api/wearable/5.5/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html [hunk 2]
```diff
@@ -251,6 +251,7 @@ <h2><a class="anchor" id="CAPI_SYSTEM_DEVICE_BATTERY_MODULE_OVERVIEW"></a>
 
 <p>Enumeration for battery property information. </p>
 <dl class="user"><dt><b>Since :</b></dt><dd>3.0 </dd></dl>
+<dl class="remark"><dt><b>Remarks:</b></dt><dd>If fuel gauge hardware does not provide average current or average voltage, CURRENT_AVERAGE/VOLTAGE_AVERAGE can be substituted by CURRENT_NOW/VOLTAGE_NOW, respectively. </dd></dl>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/api/wearable/5.5/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html]: Thanks!, fixed it

File: docs/application/native/api/mobile/6.0/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html [hunk 3]
```diff
@@ -251,6 +251,7 @@ <h2><a class="anchor" id="CAPI_SYSTEM_DEVICE_BATTERY_MODULE_OVERVIEW"></a>
 
 <p>Enumeration for battery property information. </p>
 <dl class="user"><dt><b>Since :</b></dt><dd>3.0 </dd></dl>
+<dl class="remark"><dt><b>Remarks:</b></dt><dd>If fuel gauge hardware does not provide average current or average voltage, CURRENT_AVERAGE/VOLTAGE_AVERAGE can be substituted by CURRENT_NOW/VOLTAGE_NOW, respectively. </dd></dl>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/api/mobile/6.0/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html]: Thanks!, fixed it

File: docs/application/native/api/mobile/5.5/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html [hunk 4]
```diff
@@ -251,6 +251,7 @@ <h2><a class="anchor" id="CAPI_SYSTEM_DEVICE_BATTERY_MODULE_OVERVIEW"></a>
 
 <p>Enumeration for battery property information. </p>
 <dl class="user"><dt><b>Since :</b></dt><dd>3.0 </dd></dl>
+<dl class="remark"><dt><b>Remarks:</b></dt><dd>If fuel gauge hardware does not provide average current or average voltage, CURRENT_AVERAGE/VOLTAGE_AVERAGE can be substituted by CURRENT_NOW/VOLTAGE_NOW, respectively. </dd></dl>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/api/mobile/5.5/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html]: Thanks!, fixed it

File: docs/application/native/api/iot-headless/6.0/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html [hunk 5]
```diff
@@ -149,6 +149,7 @@ <h2><a class="anchor" id="CAPI_SYSTEM_DEVICE_BATTERY_MODULE_OVERVIEW"></a>
 
 <p>Enumeration for battery health information. </p>
 <dl class="user"><dt><b>Since :</b></dt><dd>3.0 </dd></dl>
+<dl class="remark"><dt><b>Remarks:</b></dt><dd>If fuel gauge hardware does not provide average current or average voltage, CURRENT_AVERAGE/VOLTAGE_AVERAGE can be substituted by CURRENT_NOW/VOLTAGE_NOW, respectively. </dd></dl>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/api/iot-headless/6.0/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html]: Thanks!, fixed it

File: docs/application/native/api/iot-headless/5.5/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html [hunk 6]
```diff
@@ -149,6 +149,7 @@ <h2><a class="anchor" id="CAPI_SYSTEM_DEVICE_BATTERY_MODULE_OVERVIEW"></a>
 
 <p>Enumeration for battery health information. </p>
 <dl class="user"><dt><b>Since :</b></dt><dd>3.0 </dd></dl>
+<dl class="remark"><dt><b>Remarks:</b></dt><dd>If fuel gauge hardware does not provide average current or average voltage, CURRENT_AVERAGE/VOLTAGE_AVERAGE can be substituted by CURRENT_NOW/VOLTAGE_NOW, respectively. </dd></dl>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/api/iot-headless/5.5/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html]: Thanks!, fixed it

File: docs/application/native/api/iot-headed/6.0/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html [hunk 7]
```diff
@@ -251,6 +251,7 @@ <h2><a class="anchor" id="CAPI_SYSTEM_DEVICE_BATTERY_MODULE_OVERVIEW"></a>
 
 <p>Enumeration for battery property information. </p>
 <dl class="user"><dt><b>Since :</b></dt><dd>3.0 </dd></dl>
+<dl class="remark"><dt><b>Remarks:</b></dt><dd>If fuel gauge hardware does not provide average current or average voltage, CURRENT_AVERAGE/VOLTAGE_AVERAGE can be substituted by CURRENT_NOW/VOLTAGE_NOW, respectively. </dd></dl>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/6.0/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html]: Thanks!, fixed it

File: docs/application/native/api/iot-headed/5.5/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html [hunk 8]
```diff
@@ -251,6 +251,7 @@ <h2><a class="anchor" id="CAPI_SYSTEM_DEVICE_BATTERY_MODULE_OVERVIEW"></a>
 
 <p>Enumeration for battery property information. </p>
 <dl class="user"><dt><b>Since :</b></dt><dd>3.0 </dd></dl>
+<dl class="remark"><dt><b>Remarks:</b></dt><dd>If fuel gauge hardware does not provide average current or average voltage, CURRENT_AVERAGE/VOLTAGE_AVERAGE can be substituted by CURRENT_NOW/VOLTAGE_NOW, respectively. </dd></dl>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/5.5/group__CAPI__SYSTEM__DEVICE__BATTERY__MODULE.html]: Thanks!, fixed it

**Review by sgchoi5 — COMMENTED** [review]
Fixed all issues.

**Review by sagarkedari — APPROVED** [review]
@sgchoi5 Thank you for updating the feedback comments. The changes looks fine now.

**Pull request merged**